### ISSUE TITLE
Default value of `highlights` param have not been applied

### ISSUE THREAD

**Issue by Milly** [shun/ddu-source-rg#15]
The default value of `highlight` in the documentation have not actually been applied.

https://github.com/shun/ddu-source-rg/blob/9c52f4fb7d5218daddff0bacc187ac1114896046/doc/ddu-source-rg.txt#L87-L90

Therefore, an error always occurs when the highlight is not set.

```
[denops] Error: Failed to call 'call' with ["ddu#ui#ff#_update_buffer", ...(snip!)...: Vim(call):E970: Unknown highlight group name: ''
```

Why did you remove the fallback code in 714d23d?

**Comment by shun** [comment]
Thanks for making the issue.
At first, I didn't have any idea not to set `highlight` option.
Therefore, I set the default value.
However, I realized to set blank word for not to set the highlight.

So, My understanding is this is documentation bug.
And, probably this error doesn't occur by neovim.

I checked this with Neovim v0.6.1.
I also facing this error with vim 8.2.4650.

I will try to fix this.
Thank you.

**Comment by shun** [comment]
Fixed #15.

I checked vim source code.
Vim set the error for unknown group name.
Therefore, the plugin have to set the default group name.
So, set the  default value as document again.
